scss/_base_background.scss

```
@@ -1,4 +1,5 @@
 @mixin vf-base-background {
+  // stylelint-disable-next-line selector-max-type
```

I think we should comment the places where we disable it
```suggestion
  // stylelint-disable-next-line selector-max-type -- base styles can use type selectors
```